files/english/ORGANIZATION-DAY.md

```diff
@@ -1 +1,22 @@
-# Organization on the day
\ No newline at end of file
+# Organization on the day
+
+>Here are some instructions and tips to be followed by the PUG-PE organizing members and volunteers on the event's day.
+
+## Instructions
+
+### Some things to be aware of before the event's day
+
+It is important to the organizing member and volunteers to check if there are some items that might be necessary (e.g., table, power plugs, video projector). It is wanted that, in case those items are not available, the members should look for alternate solutions so the meetup can happen in the best possible way for everyone.
+
+---
+
+### Show up early
+
+The people in charge of the meetup's organization need to set up the event site. So... Show up early! This way you can solve any unexpected issues calmly, set up the chairs, the video projector, greet the speakers and the participants.
+Lots of times the meetups happen, for example, in universities classrooms. Most likely, not all participants know the place and may end up getting lost and not finding the room where the event will be held. So... Make sure there are directions to the event's site!
+
+---
+
+### PUG-PE introduction
+
+The PUG-PE is an event that is always welcoming new participants and lots of them, besides participating on the meet up, do not know the Python local community and, sometimes, not even what a community is. That's why it is really important that there is a brief introduction, lasting between 5 and 10 minutes, introducing what is a comunitty and the Python local community and what are their goals.
```

Comment:
Honestamente não sei como poderia ficar melhor, só coloquei essa sugestão por ser a primeira coisa que veio à cabeça. Mas os dois "and" me levam a querer construir essa sentença de um jeito diferente. O que tu acha?
```suggestion
The PUG-PE is an event that is always welcoming new participants and lots of them, besides participating on the meet up, do not know the Python local community and, sometimes, not even what a community is. That's why it is really important that there is a brief introduction, lasting between 5 and 10 minutes, introducing: what is a community; the Python local community; and what are their goals.
```

Reply:
@giuliocc dei uma alterada, vê o que tu achas 😛

Reply:
Muito melhor :)